src/sentry/static/sentry/app/actionCreators/savedSearches.jsx

```diff
@@ -10,14 +10,21 @@ export function fetchSavedSearches(api, orgId, useOrgSavedSearches = false) {
     data.use_org_level = '1';
   }
 
+  SavedSearchesActions.fetchSavedSearches();
```

Comment:
The first time I read this, I thought this did the XHR request. Would a name like `startFetchSavedSearches()` be less confusing?